frontend/javascripts/dashboard/dataset_view.js

```diff
@@ -151,6 +151,7 @@ function DatasetView(props: Props) {
         datasets={filteredDatasets}
         searchQuery={searchQuery}
         isUserAdmin={Utils.isUserAdmin(props.user)}
```

Comment:
I was wondering why this utils method is needed and noticed that all team managers are considered admins when using this method. At least in this context, this appears to be wrong with the new permission system. I would recommend to check all usages of this method and whether they still make sense!

For reference the utils method:
```
export function isUserAdmin(user: APIUser): boolean {
  return user.isAdmin || isUserTeamManager(user);
}
```

Reply:
I suggest to add a new function that is called `isUserAdminOrTeamManager` in addition to Ì `isUserAdmin`. Than we can do fine-grained control in all components.

Reply:
I went through all calls of `Utils.isUserAdmin` and replaced them with `Utils.isUserAdminOrTeamManager` where appropriate. see commit 2567a1b